src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/IS91DeterminationService.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonApiClient
+
+@Service
+class IS91DeterminationService(
+  private val prisonApiClient: PrisonApiClient
+) {
+
+  private object IS91Constants {
```

Comment:
meh, bit more idiomatic to make this a [companion object](https://kotlinlang.org/docs/object-declarations.html#companion-objects) now it sits inside a class